Lib/glyphsLib/builder.py

```diff
@@ -340,6 +344,10 @@ def set_custom_params(ufo, parsed=None, data=None, misc_keys=(), non_info=()):
         if name.startswith('openTypeOS2Win') and value < 0:
             value = -value
 
+        # Some Glyphs files store weightClass/widthClass as string values.
+        if name in ('openTypeOS2WeightClass', 'openTypeOS2WidthClass'):
+            value = int(value)
```

Comment:
the `casting.py` module defines `weightClass` and `widthClass` as `string`, maybe we should change that to `integer`?
https://github.com/googlei18n/glyphsLib/blob/3937aaa9866e1087b3b00826cb6c0e492cc9f098/Lib/glyphsLib/casting.py#L502-L503